internal/service/advancedclustertpf/testdata/TestAccMockableAdvancedCluster_symmetricShardedOldSchema.yaml

```diff
@@ -90,7 +90,7 @@ steps:
             text: "true"
       - path: /api/atlas/v2/groups/{groupId}/clusters
         method: POST
-        version: '2024-10-23'
+        version: '2024-08-05'
```

Comment:
@lantoli you can see the required update here

Reply:
@EspenAlbert  oh, nice!